setup.py

```diff
@@ -278,87 +300,89 @@ def run(self):
+    data_files.append(("share/man/de/man1", ["doc/de/linkchecker.1"]))
+    data_files.append(("share/man/de/man5", ["doc/de/linkcheckerrc.5"]))
+    data_files.append(
+        (
+            "share/linkchecker/examples",
+            [
+                "config/linkchecker-completion",
+                "doc/examples/check_blacklist.sh",
+                "doc/examples/check_for_x_errors.sh",
+                "doc/examples/check_urls.sh",
+            ],
+        )
+    )
 
 setup(
-    name = AppName,
-    version = AppVersion,
-    description = Description,
-    keywords = "link,url,site,checking,crawling,verification,validation",
-    author = myname,
-    author_email = myemail,
-    maintainer = myname,
-    maintainer_email = myemail,
-    url = "https://linkchecker.github.io/linkchecker/",
-    license = "GPL",
-    long_description = get_long_description(),
-    distclass = MyDistribution,
-    cmdclass = {
-        'install_lib': MyInstallLib,
-        'install_data': MyInstallData,
-        'clean': MyClean,
+    name=AppName,
+    version=AppVersion,
+    description=Description,
+    keywords="link,url,site,checking,crawling,verification,validation",
+    author=myname,
+    author_email=myemail,
+    maintainer=myname,
+    maintainer_email=myemail,
+    url="https://linkchecker.github.io/linkchecker/",
+    license="GPL",
+    long_description=get_long_description(),
+    distclass=MyDistribution,
+    cmdclass={
+        "install_lib": MyInstallLib,
+        "install_data": MyInstallData,
+        "clean": MyClean,
     },
-    packages = find_packages(include=["linkcheck", "linkcheck.*"]),
-    scripts = scripts,
-    data_files = data_files,
-    classifiers = [
-        'Topic :: Internet :: WWW/HTTP :: Site Management :: Link Checking',
-        'Development Status :: 5 - Production/Stable',
-        'License :: OSI Approved :: GNU General Public License (GPL)',
-        'Programming Language :: Python',
+    packages=find_packages(include=["linkcheck", "linkcheck.*"]),
+    scripts=scripts,
+    data_files=data_files,
+    classifiers=[
+        "Topic :: Internet :: WWW/HTTP :: Site Management :: Link Checking",
+        "Development Status :: 5 - Production/Stable",
+        "License :: OSI Approved :: GNU General Public License (GPL)",
+        "Programming Language :: Python",
         "Programming Language :: Python :: 3.5",
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
     ],
-    options = {
-    },
+    options={},
     # Requirements, usable with setuptools or the new Python packaging module.
-    python_requires = '>= 3.5',
-    install_requires = [
-        'requests >= 2.4',
-        'dnspython',
-        'beautifulsoup4',
-        'pyxdg',
-    ],
+    python_requires=">= 3.5",
+    install_requires=["requests >= 2.4", "dnspython", "beautifulsoup4", "pyxdg"],
```

Comment:
No, let's keep install_requires on separate lines for readability and future extensibility.

Reply:
problem with manual changes like this is that black will either complain or break them on the next run... if we go black, we go all the way...

Reply:
We don't have to?  We can use black to fix flake8/pycodestyle, and then not use black any more.  flake8 in the CI will prevent style regressions.

Reply:
i am fine either way

Reply:
Done.

My personal goal was to make flake8 usable on the command line and enforceable in Travis. There are too many formatting problems to fix by hand, the only other option would be to exclude a number of checks. black was a means to an end for me. Not that it couldn't be part of a workflow in the future (would be quite an adjustment) but the bugs that have shown up in the review seem to suggest that is a way off.